src/dbxdeploy/_config/config.yaml

```diff
@@ -30,6 +30,11 @@ parameters:
       force_end_file_newline: False
       black:
         enabled: True
+    databricks:
```

Comment:
I would move this section to dbxdeploy.target.package.something to be closer to offline_install: False

Reply:
Here it looks like some very general Databricks thing, not very package-generation specific.

Reply:
dbxdeploy.target.package.building?